src/mca/ras/base/ras_base_allocate.c

```diff
@@ -426,22 +425,25 @@ void prte_ras_base_allocate(int fd, short args, void *cbdata)
         if (NULL == (app = (prte_app_context_t *) prte_pointer_array_get_item(jdata->apps, i))) {
             continue;
         }
-        if (prte_get_attribute(&app->attributes, PRTE_APP_HOSTFILE, (void **) &hosts,
-                               PMIX_STRING)) {
+        if (prte_get_attribute(&app->attributes, PRTE_APP_HOSTFILE, (void **) &hosts, PMIX_STRING)) {
             PRTE_OUTPUT_VERBOSE((5, prte_ras_base_framework.framework_output,
                                  "%s ras:base:allocate adding hostfile %s",
                                  PRTE_NAME_PRINT(PRTE_PROC_MY_NAME), hosts));
 
             /* hostfile was specified - parse it and add it to the list */
-            if (PRTE_SUCCESS != (rc = prte_util_add_hostfile_nodes(&nodes, hosts))) {
-                free(hosts);
-                PRTE_DESTRUCT(&nodes);
-                /* set an error event */
-                PRTE_ACTIVATE_JOB_STATE(jdata, PRTE_JOB_STATE_ALLOC_FAILED);
-                PRTE_RELEASE(caddy);
-                return;
-            }
+            hostlist = prte_argv_split(hosts, ',');
```

Comment:
does delimiter need to include a space?  i.e., "," and " " delimiters for parsing the hostfile?  Not sure if this is change in behavior between `prte_util_add_hostfile_nodes()` and `prte_argv_split()`.

Reply:
We don't support anything but comma-delimited lists of hostfiles and hosts. The `prte_util_add_hostfile_nodes` just takes one hostfile at a time, and the revised CLI aggregates all the file names into a single comma-delimited string. So this change just splits that aggregated string and feeds them in one at a time. Before, we only allowed the user to specify a single hostfile - this removed that constraint.

Reply:
ok, i was thinking this was a list of hosts not hostfiles.